### PR TITLE
Make Worker Timeout Configurable

### PR DESCRIPTION
**Pull request by bauerj**
This pull request has been imported from jonaswinkler/paperless-ng#1471 and was originally opened by evilsidekick293 on 2021-12-05 20:35:53.

---

see #1469 

This is especially interesting for the combination of a large file and a weak hardware setup.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -507,6 +507,12 @@ PAPERLESS_THREADS_PER_WORKER=<num>
     PAPERLESS_THREADS_PER_WORKER automatically.
 
 
+PAPERLESS_WORKER_TIMEOUT=<num>
+    Machines with few cores or weak ones might not be able to finish OCR on
+    large documents within the default 1800 seconds. So extending this timeout
+    may prove be useful on weak hardware setups.
```
Comment (apeltzer) [inline docs/configuration.rst:513]:
```suggestion
    may prove to be useful on weak hardware setups.
```

**Review by apeltzer — APPROVED** [review]
Mini typo, otherwise good

**Comment by bauerj** [comment]
@apeltzer please create a new branch in your fork of the repo off evilsidekick293:feature/worker-timeout-fix and fix it there. Then you can change the head branch for this PR to your branch and have it merged 🙏

**Comment by bauerj** [comment]
Oh, thanks @evilsidekick293, that's even better!

File: docs/configuration.rst (hunk 1 above)
Comment (evilsidekick293) [reply docs/configuration.rst]: done follow-up commit

**Comment by yschroeder** [comment]
The typo is fixed. Merging.

**Comment by oxi0** [comment]
Hi, 

when I use this and change the value from 1800 to 3600, I will get following Warnings on Docker-Log:

 ```
paperless_1          |         /usr/local/lib/python3.9/site-packages/django_q/conf.py:139: UserWarning: Retry and timeout are misconfigured. Set retry larger than timeout,
paperless_1          |         failure to do so will cause the tasks to be retriggered before completion.
paperless_1          |         See https://django-q.readthedocs.io/en/latest/configure.html#retry for details.
```

Is it okay or should I change another parameter?

**Comment by yschroeder** [comment]
Can you please create an issue and reference this PR? Thank you!

**Pull request merged**